packages/webdriverio/tests/middleware.test.js

```diff
@@ -141,4 +141,20 @@ describe('middleware', () => {
             expect(waitForExist.default.mock.calls).toHaveLength(1)
         })
     })
+
+    describe('elementErrorHandler', () => {
+        it('should take last error', async () => {
```

Comment:
can you please also make sure that regex works as expected? Verify error message, I mean.